plugins/cuda/cuda_plugin.c

```diff
@@ -118,7 +118,12 @@ static int launch_cuda_checkpoint(const char **args, char *buf, int buf_size)
 
 		close_fds(STDERR_FILENO + 1);
 
-		return execvp(args[0], (char **)args);
```

Comment:
two other child "returns" have to be replaced with exit too

Reply:
Thank you for pointing this out. I have updated the patch to use `_exit()` instead of `return`.